lib/oref0-setup/basal_profile.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
Why is this basal_profile.json included. I can't find any reference of it being used. If it's meant for creating a test, please name the file accordingly.

Reply:
This is for x12 pumps: see #665